File: app/src/main/java/com/cool/eye/mvp/java/test/ITestView.java
package com.cool.eye.mvp.java.test;

import com.cool.eye.mvp.java.ILoadView;

/**
 * @author ycb
 * @date 2018/8/17
 */
public interface ITestView extends ILoadView {

    void setTest(String testMsg);

    void toAnotherPage();
}

File: app/src/main/java/com/cool/eye/mvp/kotlin/test/ITestView.kt
package com.cool.eye.mvp.kotlin.test

import com.cool.eye.mvp.kotlin.ILoadView
import com.cool.eye.mvp.kotlin.IView

/**
 * @author ycb
 * @date 2018/8/17
 */
interface ITestView : IView, ILoadView {

    fun setTest(testMsg: String)

    fun toAnotherPage(){}
}